pallets/teerex/src/lib.rs

```diff
@@ -637,6 +647,23 @@ pub mod pallet {
 			// Do not pay a fee
 			Ok(Pays::No.into())
 		}
+
+		/// This extrinsic is used to set ScheduleEnclave storage item
+		/// This storage item is used to perform feature control during register_enclave
+		/// Can only be called by the Teerex Admin
+		#[pallet::call_index(31)]
+		#[pallet::weight((195_000_000, DispatchClass::Normal, Pays::No))]
+		pub fn set_scheduled_enclave_check(
```

Comment:
Sorry for being pernickety, the name is still misleading..  you "set the scheduled-enclave check" to _true_, but in fact, the check is skipped (because you actually set the SkipScheduledEnclaveCheck storage)

How about:
```
pub fn set_skip_scheduled_enclave_check(origin, should_skip)
```

Reply:
Updated ✅